src/main/config/config_eeprom.c

```diff
@@ -78,6 +79,29 @@ typedef struct {
     uint32_t word;
 } PG_PACKED packingTest_t;
 
+#ifdef EEPROM_IN_EXTERNAL_FLASH
+bool loadEEPROMFromExternalFlash(void)
+{
+    const flashPartition_t *flashPartition = flashPartitionFindByType(FLASH_PARTITION_TYPE_CONFIG);
```

Comment:
technically yes, but not when `EEPROM_IN_EXTERNAL_FLASH` is defined as the call to `flashConfigurePartitions()` will add one.

if there's a need to add fallback storage support in future this can be addressed as part of that work.

Reply:
Not sure here ... the code can be reused/modified/applied to sdcard/ etc and eventually this will cause hardfault. Being a bit defensive here may be good idea ...